source/extensions/transport_sockets/tls/context_config_impl.cc

```diff
@@ -182,7 +182,19 @@ ContextConfigImpl::ContextConfigImpl(
                                                 default_min_protocol_version)),
       max_protocol_version_(tlsVersionFromProto(config.tls_params().tls_maximum_protocol_version(),
                                                 default_max_protocol_version)),
-      factory_context_(factory_context) {
+      factory_context_(factory_context),
+      tls_key_log_path_(config.tls_key_log_path().empty() ? "/var/log/envoy/tlskey.log"
```

Comment:
when `tls_key_log_path` is empty, should this feature being disabled?

Reply:
In my opinion, the log path should not be used to enable/disable a feature. We can just use a default path.